docs/docs/providers/asgardeo.md

```diff
@@ -0,0 +1,49 @@
+---
+id: asgardeo
+title: Asgardeo
+---
+
+## Documentation
+
+https://wso2.com/asgardeo/docs/guides/authentication
+
+
+## Instructions
+
+- Log into https://console.asgardeo.io.
+- Next, go to "Application" tab (More info: https://wso2.com/asgardeo/docs/guides/applications/register-oidc-web-app/).
+- Register standard based - Open id connect, application
+- After registring the application, go to protocol tab.
+- Check `code` grant type.
+- Add Authorized redirect URLs & Allowed origins fields.
+- Make Email, First Name, Photo URL user attributes mandatory from the console.
+
+In `.env` add the following entries:
+
+```
+NEXTAUTH_URL=<Your application url e.g: http://localhost:3000>
+SECRET=<Your secret here>
```

Comment:
Let's remove these as they are not directly related to how to set-up the provider

Reply:
Removed these env configs from the doc